### PR TITLE
updated zookeeper changes for authentication

### PR DESCRIPTION
**Pull request by anshupande**

### CODE REVIEW

**Comment by anshupande** [comment]
mesos & marathon are still up and one can hit the other. I verified it as I can see the slaves register in the Mesos UI and deployment was verified in both mesos and marathon UI CC @iderdik @chr0n1x @fraczles

File: v2/fleet-local/control/mesos-master@.service [hunk 1]
```diff
@@ -30,7 +32,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   --log_dir=/var/log/mesos \
   --quorum=3 \
   --work_dir=/var/lib/mesos/master \
-  --zk=zk://`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
+  --zk=zk://$($zk_username):$($zk_password)@`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
```
Comment (chr0n1x) [inline v2/fleet-local/control/mesos-master@.service]: let's take this chance to move the `etcdctl get /environment/ZOOKEEPER_ENDPOINT` into an `Environment`, the same way that you have `zk_username` & `zk_password`

File: v2/fleet-local/worker/mesos-slave@.service [hunk 2]
```diff
@@ -41,7 +43,7 @@ ExecStart=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && \
     --executor_registration_timeout=10mins \
     --hostname=`curl -s http://169.254.169.254/latest/meta-data/public-hostname` \
     --log_dir=/var/log/mesos \
-    --master=zk://`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos \
+    --master=zk://$($zk_username):$($zk_password)@`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos \
```
Comment (chr0n1x) [inline v2/fleet-local/worker/mesos-slave@.service]: same here, for consistency

**Comment by chr0n1x** [comment]
:+1:

**Comment by fraczles** [comment]
:+1:

**Comment by anshupande** [comment]
Before I merge this I will add zk username and password in secrets and upload the same to AWS for each account

**Comment by anshupande** [comment]
zk username and passwords have been updated in secrets and as well uploaded to AWS accounts: https://secrets.cloudops.adobe.com/SecretView.aspx?secretid=21521

**Pull request merged**